CHANGELOG.md

```diff
@@ -3,6 +3,13 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+- Support for partial reads on beamfits files. This can be done on frequency, azimuth
+  or zenith angle (if beam is in az/za coordinates).
+- Logging statements (INFO level) in `UVdata.check`.
+### Changed
+- Performance improvement when doing a `select` using the `bls` parameter (~70%
+  improvement in the `_select_preprocess` function)
```

Comment:
maybe add something to make it clear this is on the UVData object?

Reply:
Done.

Reply:
I think you forgot to push after you did it?